### ISSUE TITLE
我想训练一个支持空格的英文识别模型，但是只要在数据集中加入带空格的数据迭代几次后就出现loss为nan了

### ISSUE THREAD

**Issue by Roy2008** [WenmuZhou/PytorchOCR#171]
如题，只要在数据集中加入带空格的数据迭代几次后就出现loss为nan，有没有遇到过这个问题，不知道怎么解决了

![图片](https://user-images.githubusercontent.com/46923766/115850622-a4915f80-a458-11eb-8c8a-68ca3ebc1c58.png)

**Comment by Roy2008** [comment]
OH   我知道了  是n_class设置错误

**Comment by nissansz** [comment]
我的也出现这个nan错误了，n_class是用字典行数+1,没问题吧？怎么解决？

2023-01-26 20:50:00,587 - torchocr - INFO - Training...
2023-01-26 20:50:00,587 - torchocr - INFO - train dataset has 3615327 samples,225958 in dataloader
2023-01-26 20:50:00,588 - torchocr - INFO - eval dataset has 20000 samples,5000 in dataloader
2023-01-26 20:50:34,076 - torchocr - INFO - [0/200] - [10/225958] - lr:0.001 - loss:147.4167 - acc:0.0000 - norm_edit_dis:0.0000 - time:33.4870
2023-01-26 20:51:04,685 - torchocr - INFO - [0/200] - [20/225958] - lr:0.001 - loss:271.0975 - acc:0.0000 - norm_edit_dis:0.0000 - time:30.6092
2023-01-26 20:51:33,723 - torchocr - INFO - [0/200] - [30/225958] - lr:0.001 - loss:125.6765 - acc:0.0000 - norm_edit_dis:0.0000 - time:29.0379
2023-01-26 20:52:03,072 - torchocr - INFO - [0/200] - [40/225958] - lr:0.001 - loss:104.1359 - acc:0.0000 - norm_edit_dis:0.0000 - time:29.3491
2023-01-26 20:52:31,204 - torchocr - INFO - [0/200] - [50/225958] - lr:0.001 - loss:66.1327 - acc:0.0000 - norm_edit_dis:0.0000 - time:28.1294
2023-01-26 20:53:04,027 - torchocr - INFO - [0/200] - [60/225958] - lr:0.001 - loss:41.0694 - acc:0.0000 - norm_edit_dis:0.0000 - time:32.8223
2023-01-26 20:53:35,120 - torchocr - INFO - [0/200] - [70/225958] - lr:0.001 - loss:22.6170 - acc:0.0000 - norm_edit_dis:0.0000 - time:31.0934
2023-01-26 20:54:06,949 - torchocr - INFO - [0/200] - [80/225958] - lr:0.001 - loss:11.2823 - acc:0.0000 - norm_edit_dis:0.0000 - time:31.8285
2023-01-26 20:54:42,005 - torchocr - INFO - [0/200] - [90/225958] - lr:0.001 - loss:8.6058 - acc:0.0000 - norm_edit_dis:0.0000 - time:35.0553
2023-01-26 20:55:11,051 - torchocr - INFO - [0/200] - [100/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:29.0454
2023-01-26 20:55:41,430 - torchocr - INFO - [0/200] - [110/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:30.3788
2023-01-26 20:56:12,583 - torchocr - INFO - [0/200] - [120/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:31.1517
2023-01-26 20:56:43,814 - torchocr - INFO - [0/200] - [130/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:31.2295
2023-01-26 20:57:16,205 - torchocr - INFO - [0/200] - [140/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:32.3914
2023-01-26 20:57:46,468 - torchocr - INFO - [0/200] - [150/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:30.2631
2023-01-26 20:58:21,284 - torchocr - INFO - [0/200] - [160/225958] - lr:0.001 - loss:nan - acc:0.0000 - norm_edit_dis:0.0000 - time:34.8160
